neural-plasticity.csl

```diff
@@ -0,0 +1,145 @@
+    <link href="http://www.zotero.org/styles/neural-plasticity" rel="self"/>
+    <author>
+      <name>Lucio Marinelli</name>
+      <email>lucio.marinelli@unige.it</email>
+      <uri>http://www.luciomarinelli.com</uri>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <eissn>1687-5443</eissn>
+    <updated>2017-04-28T14:08:20+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <group>
+      <names variable="author">
+        <name and="text" initialize-with="."/>
+        <label form="short" prefix=" " strip-periods="true"/>
+        <substitute>
+          <names variable="editor"/>
+        </substitute>
+      </names>
+    </group>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name initialize-with="." delimiter=", " and="text" name-as-sort-order="all" sort-separator=", "/>
+      <label form="short" prefix=" (" suffix=")"/>
+    </names>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+        <text variable="title" quotes="true"/>
+      </if>
+      <else>
+        <text variable="title" quotes="true" suffix=","/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <text variable="publisher" font-style="italic"/>
+    <text variable="publisher-place" prefix=", "/>
+  </macro>
+  <macro name="page">
+    <group>
+      <label plural="always" suffix=" " variable="page" form="short"/>
+      <text variable="page"/>
+    </group>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="1" collapse="citation-number">
+    <sort>
+      <key variable="citation-number"/>
+    </sort>
+    <layout prefix="[" suffix="]" delimiter=", ">
+      <text variable="citation-number"/>
+    </layout>
+  </citation>
+  <bibliography entry-spacing="0" second-field-align="flush">
+    <layout suffix=".">
+      <text variable="citation-number" prefix="[" suffix="]"/>
+      <text macro="author" suffix=", "/>
+      <choose>
+        <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+          <group delimiter=" ">
+            <text macro="title" suffix="."/>
+            <text macro="publisher"/>
+            <date variable="issued" prefix=", ">
+              <date-part name="year"/>
+            </date>
+          </group>
+        </if>
+        <else-if type="article-journal">
+          <group delimiter=" ">
+            <text macro="title" suffix="."/>
+            <text variable="container-title" font-style="italic" suffix="."/>
+            <text variable="volume" prefix="vol. " suffix=","/>
+            <number prefix="no. " suffix="," variable="issue"/>
+            <text variable="page" form="short" prefix="pp. " suffix=","/>
```

Comment:
I applied your suggestions but I don't know how to use a label for pages instead of having pp. hardcoded...